src/components/entries/FEEL/FeelPopup.js

```diff
@@ -1,304 +0,0 @@
-import {
-  useCallback,
-  useEffect,
-  useRef,
-  useState
-} from 'preact/hooks';
-
-import { FeelPopupContext } from './context';
-
-import { usePrevious } from '../../../hooks';
-
-import { Popup } from '../../Popup';
-
-import CodeEditor from './FeelEditor';
-
-import TemplatingEditor from '../templating/TemplatingEditor';
-import { LaunchIcon } from '../../icons';
-
-export const FEEL_POPUP_WIDTH = 700;
-export const FEEL_POPUP_HEIGHT = 250;
-
-
-/**
- * FEEL popup component, built as a singleton. Emits lifecycle events as follows:
- *  - `feelPopup.open` - fired before the popup is mounted
```

Comment:
We have removed these events which is technically a breaking change.

Reply:
These events are being used in the Desktop Modeler: https://github.com/camunda/camunda-modeler/blob/develop/client/src/app/tabs/bpmn/modeler/features/properties-panel-keyboard-bindings/FeelPopupKeyboardBindings.js

Reply:
I've added comments so we don't forget about this: https://github.com/bpmn-io/properties-panel/pull/409/commits/ae1df41e3d129934e236700fd1f1e3ff5e464caf

The issue is that even if the default renderer fires these events (they are actually fired by the component that is rendered) there is no guarantee that a custom render will fire these events.